src/shared/canvas/MyData/MyData.tsx

```diff
@@ -116,6 +76,7 @@ const HomeMyData: React.FC<{}> = () => {
       ? `${dateField}-${dateFilterType}-${dateFilterRange}`
       : undefined;
   const order = sort.join('-');
+  const resourceTypes = types?.map(item => get(item, 'value'));
```

Comment:
I think `types` is always defined now. Atleast I don't see any place where it is being set to to a falsy value. So, prehaps we can remove the `?` check here and update the TS types for it.

Reply:
it's not always there, in the query, if you want to not include some fields in the `queryParams` you have to make them `undefined`, and this is why the types must be `undefined`